sdks/python/apache_beam/testing/util.py

```diff
@@ -113,6 +134,7 @@ def assert_that(actual, matcher, label='assert_that', reify_windows=False):
       expectations and raises BeamAssertException if they are not met.
     label: Optional string label. This is needed in case several assert_that
       transforms are introduced in the same pipeline.
+    windowed: If True, matcher is passed a tuple (element, window to check).
```

Comment:
<!--new_thread; commit:1d8257d6902c54d33062c23cce99e06fbf42c41d; resolved:0-->
Name of the argument is `windowing`. Either change the name of the argument or update the documentations. (`windowed` sounds like a better name to me but it is your call.)

Reply:
Oops! I forgot to update that. Now that you make me think, I am calling it custom_windowing, because it reads better in lines like:
`pcoll | WindowInto(custom_windowing or window.GlobalWindows())`